README.md

```diff
@@ -57,7 +57,7 @@ PXLAlbumFilterOptions filterOptions = new PXLAlbumFilterOptions();
 filterOptions.minTwitterFollowers = 1000;
 filterOptions.minInstagramFollowers = 2000;
 PXLAlbumSortOptions sortOptions = new PXLAlbumSortOptions();
-sortOptions.sortType = PXLAlbumSortType.PHOTORANK;
+sortOptions.sortType = PXLAlbumSortType.DYNAMIC;
```

Comment:
didnt have a better place to put this but confirming that default sort is DESC, and recency ?

Reply:
yes https://github.com/pixlee/android-sdk/blob/master/pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLAlbumSortOptions.java#L11